_includes/rest/users.md

```diff
@@ -613,3 +611,47 @@ For example, if you want the user with id `"3KmCvT7Zsb"` to have read and write
 ```
 
 If you want to access your data ignoring all ACLs, you can use the master key provided on the Dashboard. Instead of the `X-Parse-REST-API-Key` header, set the `X-Parse-Master-Key` header. For backward compatibility, you can also do master-level authentication using HTTP Basic Auth, passing the application id as the username and the master key as the password. For security, the master key should not be distributed to end users, but if you are running code in a trusted environment, feel free to use the master key for authentication.
+
+## Impersonating a user
+
+An application may have use cases where an operator or automated system needs to
+take action on a user's behalf, under the user's authority, without access to
+the user's credentials. The Parse REST API supports these use cases with the
+`/loginAs` endpoint. This endpoint takes a `userId` parameter, and must be
+called using the master key. It will create a session for the given user ID, and
+will return the same response format as the `/login` endpoint. The new session
+will have a `createdWith` `action` value of `login`, and an `authProvider` value of
+`masterkey`.
+
+<div class="language-toggle">
+<pre><code class="bash">
+curl -X GET \
+  -H "X-Parse-Application-Id: <span class="custom-parse-server-appid">${APPLICATION_ID}</span>" \
+  -H "X-Parse-REST-API-Key: <span class="custom-parse-server-restapikey">${REST_API_KEY}</span>" \
+  -H "X-Parse-Master-Key: ${MASTER_KEY}" \
+  -H "X-Parse-Revocable-Session: 1" \
+  -G \
+  --data-urlencode 'userId=abc123' \
+  <span class="custom-parse-server-protocol">https</span>://<span class="custom-parse-server-url">YOUR.PARSE-SERVER.HERE</span><span class="custom-parse-server-mount">/parse/</span>loginAs
+</code></pre>
+<pre><code class="python">
+import json,httplib,urllib
+connection = httplib.HTTPSConnection('<span class="custom-parse-server-url">YOUR.PARSE-SERVER.HERE</span>', 443)
+params = urllib.urlencode({"userId":"abc123"})
+connection.connect()
+connection.request('GET', '<span class="custom-parse-server-mount">/parse/</span>loginAs?%s' % params, '', {
+       "X-Parse-Application-Id": "<span class="custom-parse-server-appid">${APPLICATION_ID}</span>",
+       "X-Parse-REST-API-Key": "<span class="custom-parse-server-restapikey">${REST_API_KEY}</span>",
+       "X-Parse-Master-Key": "${MASTER_KEY}",
+       "X-Parse-Revocable-Session": "1"
+     })
+result = json.loads(connection.getresponse().read())
+print result
+</code></pre>
+</div>
+
+The `/loginAs` endpoint does not run the `beforeLogin` or `afterLogin` hooks
+that would be invoked when calling the `/login` endpoint.
```

Comment:
I think we can omit to describe what other endpoints do, since they are not part of this chapter and we essentially duplicate information. We can add a link to the triggers and write this more concise as:

> Calling this endpoint does not invoke the [session triggers](https://docs.parseplatform.org/cloudcode/guide/#session-triggers) such as beforeLogin and afterLogin.